### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by docslip70** [community/community#40314] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

To do with the arcwelder plugin for 3d printers namely console version ArcWelder32_1.1.1,
This program runs correctly in my win 10 -32 bit machine but will not run in my XP 32 bit machine. The xp system barks back that it is not a valid 32 bit application.
.
We are all aware xp is dead and buried but not so for the small business who has a tall investment in his cnc routers , controllers and software. He cannot, in this case, afford to upgrade his working system.

Perhaps there's a net framework version that should be used?

Any helpful ideas will be appreciated.

Thank you

**Comment by Akash1134** [comment]
Hi @docslip70  ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
